dataikuapi/dss/notebook.py

```diff
@@ -1,4 +1,5 @@
 from .discussion import DSSObjectDiscussions
+from .notebookcontent import DSSNotebookContent
 
 class DSSNotebook(object):
```

Comment:
Suggestion.... Actually we may just revert DSSNotebook as it was in DSS 8.0 and declare it as deprecated (as well as all methods returning DSSNotebook instances).

We would create a DSSJupyterNotebook class without the `get_state()` method.

WDYT?

Reply:
Teah, let's do it that way, the legacy stuff is too hard to maintain.